src/index.html

```diff
@@ -13,6 +13,52 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
```

Comment:
The class name `stars--0` doesn't provide a clear understanding of what it represents. Consider renaming it to something more descriptive, like `stars--none`.